src/mame/machine/toaplan_gxl.h

```diff
@@ -0,0 +1,66 @@
+// license:BSD-3-Clause
+// copyright-holders:Quench
+#ifndef MAME_MACHINE_TOAPLAN_GXL_H
+#define MAME_MACHINE_TOAPLAN_GXL_H
+
+#pragma once
+
+#include "cpu/tms32010/tms32010.h"
+
+typedef device_delegate<void (u32 &main_ram_seg, u32 &dsp_addr, u16 data)> dsp_addr_cb_delegate;
+typedef device_delegate<bool (u32 main_ram_seg, u32 dsp_addr, u16 &data)> dsp_read_cb_delegate;
+typedef device_delegate<bool (u32 main_ram_seg, u32 dsp_addr, bool &dsp_execute, u16 data)> dsp_write_cb_delegate;
+
```

Comment:
These shouldn’t be in the global namespace – they can be scoped inside the device class.